### PR TITLE
Update error message 7

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-752.

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: ERROR_MESSAGES.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 4 : CT - Unexpected exception occurred during encoding. Please contact the Service Center for assistance via phone at 1-866-288-8292 or TTY: 1-877-715-6222, or by emailing QPP@cms.hhs.gov
 * 5 : CT - The file is not a QRDA-III XML document. Please ensure that the submission complies with the `(Submission year's)` implementation guide. `(Implementation guide link)`
 * 6 : CT - The measure reference results must have a single occurrence of the recognized measure GUID `(Provided measure id)` is invalid. Did you intend to send one of these `(Valid measure id suggestions)`?
-* 7 : CT - The measure reference results must have at least one measure
+* 7 : CT - TThe measure reference results must have at least one measure. Please review the measures section of your file as it cannot be empty.
```
Comment (saquino0827) [inline ERROR_MESSAGES.md]: `Tthe` is misspelled

Comment (omnivion) [reply ERROR_MESSAGES.md]: It was just a consequence of the build - no error messages were actually changed with this. Want that done in this pr or create another issue?

Comment (omnivion) [reply ERROR_MESSAGES.md]: Oh wait I was looking at the wrong PR, ignore all that from my last comment

Comment (omnivion) [reply ERROR_MESSAGES.md]: Fixed

**Pull request merged**